sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -1419,3 +1419,32 @@ def process(self, an_int):
   p.run().wait_until_finish()
 
   # [END AccessingValueProviderInfoAfterRunSnip1]
+
+  # [START CustomSessionWindow1]
+  def assign(self, context):
+    timestamp = context.timestamp
+    return [IntervalWindow(timestamp, timestamp + self.gap_size)]
+
+  # [END CustomSessionWindow1]
+
+  # [START CustomSessionWindow2]
+  def assign(self, context):
+    timestamp = context.timestamp
+
+    try:
+      gap = Duration.of(context.element[1][“gap”])
+      except:
```

Comment:
except needs to be unindented to match try.

Reply:
Python version removed.